### ISSUE TITLE
fix fr IntermediateNodes used for Measurements

### ISSUE THREAD

**Issue by VladimirAlexiev** [dbpedia/mappings-tracker#42]
@jplu
Eg http://mappings.dbpedia.org/index.php?title=Mapping_fr:Infobox_Ville_de_Moldavie&action=edit (and I'm sure many more)

```
        {{IntermediateNodeMapping
        | nodeClass = Area
        | correspondingProperty = wholeArea
        | mappings =
           {{PropertyMapping | templateProperty = superficie | ontologyProperty = value }} 
        }}
        {{IntermediateNodeMapping
        | nodeClass = Altitude
        | correspondingProperty = altitude
        | mappings =
           {{PropertyMapping | templateProperty = altitude | ontologyProperty = value }}
        }}
```

Why would you make intermediate nodes instead of using a literal property directly?

**Comment by jplu** [comment]
Hi @VladimirAlexiev 

I use an intermediateNode to represent an area because sometimes there is other values accorded to this area than just the value, like the rank for example.

I use an intermediateNode to represent an altitude because sometimes it is not a simple value, it is a scale, like for example, alt mini and alt maxi.

**Comment by VladimirAlexiev** [comment]
Why take something simple and make it complex, because in other templates a more complex variant may be needed?

http://mappings.dbpedia.org/index.php/Mapping_en:Infobox_ski_area was made in 2010 and uses minimumElevation and maximumElevation (but the difference between the two is missing).
Your mapping http://mappings.dbpedia.org/index.php/Mapping_fr:Infobox_Station_de_ski is made in 2013 and uses a completely different technique.

The whole purpose of the mapping wiki is to eliminate data differences between languages, so having different mappings won't do.

**Comment by jplu** [comment]
Indeed, I will remove the intermediate node to use `minimumElevation` and `maximumElevation` instead.

**Comment by jplu** [comment]
The mapping on http://mappings.dbpedia.org/index.php/Mapping_fr:Infobox_Station_de_ski seems better for you ?

**Comment by VladimirAlexiev** [comment]
Yes, but it still uses "takePlace" instead of pointing to its parent places directly.
And please add a few test cases

**Comment by jplu** [comment]
I replaced the usage of `takePlace` by `isPartOf`
